docs/competency-api/audience.md

```diff
@@ -4,4 +4,37 @@ title: Audience
 nav_order: 2
 parent: Competency API
 ---
-# Audience
\ No newline at end of file
+# Audience
+## Update Audience
+
+### Definition
+```
+http://api.competency-constructor.clark.center/competencies/:competencyId/audience
+```
+
+## Headers
+```
+{
+    Bearer: $BEARER-TOKEN
```

Comment:
instead of $BEARER-TOKEN can we make it like "Bearer Example.bearer.token" just to show what its supposed to look like